src/helpers/shareHelper.js

```diff
@@ -6,6 +6,9 @@ import { QUERY_TYPES } from '../queries';
 import { momentFormat } from './momentHelper';
 
 // https://facebook.github.io/react-native/docs/share
+/**
+ * @param {{ title?: string; message: string; } | { title?: string; url: string; }} shareContent
+ */
```

Comment:
`|` This operator performs a bitwise OR operation, so what is the differences with` ||`

why is it conditional now:  title + message or title + url ?
and why types are commented out , is this logic still work in progress ?

Reply:
https://jsdoc.app/about-getting-started.html

it is neither a bitwise or, not an or. it is a union type.

this is not work in progress either, and it is right that it is in the form of this comment (that is how jsdoc works)

Reply:
we also use it here, for example: https://github.com/ikuseiGmbH/smart-village-app-app/blob/master/src/helpers/momentHelper.js#L4